pkg/config/config.go

```diff
@@ -350,8 +354,8 @@ func (m *Manager) setDefaults(prefix string, flags *pflag.FlagSet, config interf
 				continue
 			}
 
-			if c, ok := face.(Configurable); ok {
-				val := fmt.Sprintf("%v", c)
+			if isConfigurableType(field.Type) {
+				val := fmt.Sprintf("%v", face)
```

Comment:
Not sure what this is supposed to do, but there's no reason to pass `Configurable` here, since `val` will always be equal to return value when just passing `face`

Reply:
We really need to start documenting our code better 😝